packages/rpc-graphql/README.md

```diff
@@ -3,61 +3,1160 @@
+you might _only_ want specifically any account that matches `MintAccount`.
+
+```typescript
+const maybeMintAddresses = [
+    'J7iup799j5BVjKXACZycYef7WQ4x1wfzhUsc5v357yWQ',
+    'JAbWqZ7S2c6jomQr8ofAYBo257bE1QJtHwbX1yWc2osZ',
+    '2AQ4CSNu6zNUZsUq4aLNUSjyrLv4qFFXQuKs5RTHbg2Y',
+    'EVW3CoyogapBfQxBFFEKGMM1bn3JyoFiqkAJdw3FHX1b',
+];
+
+const mintAccounts = [];
+
+const source = `
+    query myQuery($address: String!) {
+        account(address: $address) {
+            ... on MintAccount {
+                data {
+                    parsed {
+                        info {
+                            decimals
+                            isInitialized
+                            mintAuthority
+                            supply
+                        }
+                    }
+                }
+            }
+        }
+    }
+`;
+
+for (const address of maybeMintAddresses) {
+    const mint = await rpcGraphQL.query(source, { address });
+    if mint {
+        mintAccounts.push(mint.data.account.data.parsed.info)
+    }
+}
+```
+
+Maybe you want to handle both mints _and_ token accounts.
+
+```typescript
+const mintOrTokenAccountAddresses = [
+    'J7iup799j5BVjKXACZycYef7WQ4x1wfzhUsc5v357yWQ',
+    'JAbWqZ7S2c6jomQr8ofAYBo257bE1QJtHwbX1yWc2osZ',
+    '2AQ4CSNu6zNUZsUq4aLNUSjyrLv4qFFXQuKs5RTHbg2Y',
+    'EVW3CoyogapBfQxBFFEKGMM1bn3JyoFiqkAJdw3FHX1b',
+];
+
+const mintAccounts = [];
+const tokenAccounts = [];
+
+const source = `
+    query myQuery($address: String!) {
+        account(address: $address) {
+            ... on MintAccount {
+                data {
+                    parsed {
+                        info {
+                            decimals
+                            isInitialized
+                            supply
+                        }
+                        type
+                    }
+                }
+            }
+            ... on TokenAccount {
+                data {
+                    parsed {
+                        info {
+                            isNative
+                            mint
+                            state
+                        }
+                        type
+                    }
+                }
+            }
+        }
+    }
+`;
+
+for (const address of mintOrTokenAccountAddresses) {
+    const mintOrTokenAccount = await rpcGraphQL.query(source, { address });
+    if mintOrTokenAccount {
+        const {
+            data: {
+                account: {
+                    data: {
+                        parsed: {
+                            info: accountParsedInfo,
+                            type: accountType,
+                        }
+                    }
+                }
+            }
+        } = mintOrTokenAccount;
+        if accountType = 'mint' {
+            mintAccounts.push(accountParsedInfo)
+        } else {
+            tokenAccounts.push(accountParsedInfo)
+        }
+    }
```

Comment:
Nitpick: there's a few issues with the TS syntax. `if (mintOrTokenAccount)` and `if (accountType === 'mint')`
This looks copy-pastable so people probably will!